app/controllers/DraftMetadataChecksResultsController.scala

```diff
@@ -18,7 +21,8 @@ class DraftMetadataChecksResultsController @Inject() (
     val keycloakConfiguration: KeycloakConfiguration,
     val frontEndInfoConfiguration: ApplicationConfig,
```

Comment:
We are not using `frontEndInfoConfiguration` you can remove it from the controller and html page as well